index.bs

```diff
@@ -3362,8 +3450,9 @@ The <code>ExceptionDetails</code> type represents a JavaScript exception.
    this data with regex or something equally bad.
 
 1. Let |exception| be the result of [=serialize as a remote value=] given
-   |record|.\[[Value]], <code>1</code> as |max depth|, <code>true</code> as |node details|,
-   a <code>new set</code> as |set of known objects| and |realm|.
+   |record|.\[[Value]], <code>1</code> as |max depth|, <code>root</code> as
```

Comment:
This shouldn't be hardcoding `ownership type` (or `max depth`, but that's an existing bug); instead it should be passed from the caller and match the command parameters.

Reply:
valid point. Fixed.